PROVESFlightControllerReference/test/yamcs/test_yamcs_noop.py

```
@@ -0,0 +1,74 @@
+    return False
+
+
+def test_noop_round_trip(yamcs_client, yamcs_processor):
+    """Issue CMD_NO_OP via YAMCS and assert a NoOpReceived event comes back.
+
+    Retries the command every RETRY_INTERVAL_S until TOTAL_TIMEOUT_S elapses
+    or the event is observed, mirroring GDS conftest's start_gds fixture.  A
+    single TC can be dropped if FSW is mid-boot or in safe mode entry, so we
+    keep nudging it until either the link comes up or we hit the hard ceiling.
+    """
+    events: "queue.Queue" = queue.Queue()
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)